packages/components/src/components/PreferencePage.tsx

```diff
@@ -1,12 +1,14 @@
 import { Footer, Header, PreferenceList } from "@trycourier/react-preferences";
 import React from "react";
 
-const PreferencePage: React.FunctionComponent = () => {
+const PreferencePage: React.FunctionComponent<{ draft?: boolean }> = ({
+  draft = false,
+}) => {
   return (
     <div>
-      <Header />
-      <PreferenceList />
-      <Footer />
+      <Header draft={draft} />
+      <PreferenceList draft={draft} />
+      <Footer draft={draft} />
```

Comment:
how's draft changing footer and header? or you are just making sure you add support for it in future?

Reply:
The header and footer have a use the `showCourierFooter` field from the preferences hook so they need the draft field.